src/data_management/timeseries/locations/LocationForm.tsx

```diff
@@ -80,91 +40,52 @@ const LocationFormModel = (props:Props & PropsFromDispatch & RouteComponentProps
       {
         lat: currentRecord.geometry.coordinates[1],
         lng: currentRecord.geometry.coordinates[0],
-      };
-    const geometryRelatedAsset = 
-      relatedAsset && 
-      relatedAsset.view && 
-      relatedAsset.view[1] !== undefined && 
-      relatedAsset.view[0] !== undefined &&
-      {
-        lat: relatedAsset.view[1],
-        lng: relatedAsset.view[0],
-      };
```

Comment:
I am not covering the geometry of related asset for now and get the geometry directly from the location to simplify this a bit. I will discuss with Lex further about it.